akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/ActiveActiveShardingDirectReplication.scala

```diff
@@ -15,7 +15,7 @@ import akka.annotation.InternalApi
 import akka.persistence.typed.PublishedEvent
 import akka.persistence.typed.ReplicaId
 
-import scala.collection.JavaConverters._
+import akka.util.ccompat.JavaConverters._
```

Comment:
Strange indeed, a quick search shows we use these all over the place so there is something special how we use them here?

Reply:
A lot of the other usages are in the multi jvm tests where we don't have strictness